etc/kayobe/kolla.yml

```diff
@@ -135,10 +127,10 @@ kolla_build_blocks:
         && pip install -U git+https://github.com/stackhpc/monasca-log-api@stackhpc/log-query-api
   sahara_api_footer: |
     # Bring in hack to the vanilla plugin
-    RUN pip install -U --no-deps git+https://github.com/stackhpc/sahara.git@stackhpc/7.0.0.0b2.1
+    RUN pip install -U --no-deps git+https://github.com/stackhpc/sahara.git@stackhpc/7.0.2.1
   sahara_engine_footer: |
     # Bring in hack to the vanilla plugin
-    RUN pip install -U --no-deps git+https://github.com/stackhpc/sahara.git@stackhpc/7.0.0.0b2.1
+    RUN pip install -U --no-deps git+https://github.com/stackhpc/sahara.git@stackhpc/stackhpc/7.0.2.1
```

Comment:
I know you love StackHPC, but one is sufficient here

Reply:
Nice spot - damnit!